psalm.xml

```diff
@@ -3,7 +3,7 @@
     errorLevel="4"
     phpVersion="8.2"
     findUnusedBaselineEntry="true"
-    findUnusedCode="true"
+    findUnusedCode="false"
```

Comment:
For some reason, Psalm starts to detect classes used by `DoctrineMigrationsExtensionTest` as "unused" after my change. My motivation to investigate that is rather limited, I must admit. 😓